collection/create.go

```diff
@@ -20,11 +19,8 @@ func (dc *Collection) Create(options types.QueryOptions) (*types.AckResponse, er
 	res := <-ch
 
 	if res.Error != nil {
-		return nil, res.Error
+		return false, res.Error
 	}
 
-	ack := &types.AckResponse{}
-	json.Unmarshal(res.Result, ack)
-
-	return ack, nil
+	return true, nil
```

Comment:
Is it sure the result is not important here, can it be that we have acknowledge set to false without an error ?

Reply:
I don't know if Kuzzle/Elasticsearch can return anything else than `true` if a request is correctly formatted.

Either `false` can be returned and we have to use the value returned in the KuzzleResult structure, or it can never be false, and we may aswell remove the boolean return value completely.

Reply:
This remark is to be applied to any other function returning a boolean instead of a `AckResponse` structure